signer/tests/integration/main.rs

```diff
@@ -7,6 +7,7 @@ mod block_observer;
 mod communication;
 mod complete_deposit;
 mod contracts;
+mod devenv;
```

Comment:
What would you think of creating a separate `devenv` folder next to `integration` and putting devenv-targeted tests there? That way you don't need to ignore them and you'd just use `--test devenv` instead of `--test integration` when you want to run them? I'm just thinking that while yeah they're integration tests, they're quite special and will never run in CI etc. (well, at least until we have a working solution for CI).

Reply:
Yeah that makes sense, moved in [635bbf7](https://github.com/stacks-sbtc/sbtc/pull/1585/commits/635bbf7b55ebc739abb16ae381f2809a7dfd5baf)